packages/casting/src/follower-cosmjs.js

```diff
@@ -112,290 +145,442 @@ export const makeCosmjsFollower = (
   const leader =
     typeof leaderOrMaker === 'function' ? leaderOrMaker() : leaderOrMaker;
 
+  const tendermintClientPs = new Map();
+  /**
+   * @param {string} endpoint
```

Comment:
It can be, but in this case, the return type is safely inferred. I’m inclined to keep internal function return types implicit. They do get checked on the caller side.